.pre-commit-config.yaml

```diff
@@ -16,22 +16,6 @@ repos:
         stages: [commit]
       - id: trailing-whitespace      # Trims trailing whitespace.
 
-
-  - repo: https://github.com/timothycrosley/isort
```

Comment:
Thanks for the catch on pre-commit. My VSCode doesn't search in this file. Need to fix that.